services/ui-src/src/components/forms/AdminDashSelector.tsx

```diff
@@ -51,7 +53,7 @@ export const AdminDashSelector = ({ verbiage }: Props) => {
     selectedReport = selectedReport.replace("report-", "").toLowerCase();
     localStorage.setItem("selectedReportType", selectedReport);
 
-    if (userIsAdmin || userIsReadOnly) {
+    if (userCanReadOnly) {
```

Comment:
Comment: I'm on the fence with ones like this -- it might be more clear to keep it as the two explicitly noted groups.

Reply:
^^ This applies to all similar spots where we collapse the user groups `userIsAdmin` and `userIsReadOnly` (which for the uninitiated are composed of [super admins (bo reps) and approvers] and [help desk and internal users] respectively) down to `userCanReadOnly`.